nginx/default.template

```diff
@@ -15,9 +15,25 @@ server {
   # Long timeouts are necessary for extended split reports
   proxy_read_timeout 300s;
 
+  location ~ /districtmapping/plan/\d*/unlockedgeometries/ {
```

Comment:
Should the `\d*` here be `\d+`? I believe that the latter is for 1 or more, whereas the former is 0 or more (which would be something like `/districtmapping/plan//unlockedgeometries/`).